core/src/validator.rs

```diff
@@ -623,6 +629,29 @@ impl Validator {
 
         let start_time = Instant::now();
 
+        if !ledger_path.is_dir() {
+            return Err(anyhow!(
+                "ledger directory does not exist or is not accessible: {ledger_path:?}"
+            ));
+        }
+        let genesis_config = load_genesis(config, ledger_path)?;
+        metrics_config_sanity_check(genesis_config.cluster_type)?;
+
+        // Measure the PoH hash rate as early as possible to minimize
+        // contention with other threads
+        //
+        // Skip the check for Development clusters as this will be the type for
+        // tests or for clusters that we don't know what the rate will be
+        // without a rebuilt Bank
```

Comment:
isn't the real reason that we don't want to pay the time cost for local cluster tests and such?
Because we end up pulling the rate from the root bank later

Reply:
Kind of, a couple things a
- Unit tests are run in debug mode ... doing 10M hashes in debug mode on local machine, potentially for multiple tests in parallel is not a good time
- You're correct that we pull the value from the Bank later, and because of that, we don't necessarily want to spend the time to hash 10M hashes (in hopefully a second) if the genesis config dictates we only need to be able to hit 100k.

I can look at tweaking the comment a bit more